src/node_debug_options.cc

```diff
@@ -77,9 +77,11 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
   if (option_name == "--inspect") {
     enable_inspector = true;
-  } else if (option_name == "--inspect-brk") {
+  } else if (option_name == "--inspect-brk" || option_name == "--debug-brk") {
```

Comment:
This would also require a deprecation warning to be emitted at startup when `--debug-brk` is used. The CTC discussion also centered on adding `--debug` back as an alias for `--inspect`, also with a deprecation warning.

Note: The deprecation warning would need to use the `process.emitWarning()` mechanism. I believe there is a deprecation code already allocated for this.